cmd/livepeer/livepeer.go

```diff
@@ -269,13 +269,21 @@ func main() {
 	if *transcoder {
 		core.WorkDir = *datadir
 		if *nvidia != "" {
+			// Get a list of device ids
+			devices, err := common.ParseNvidiaDevices(*nvidia)
+			if err != nil {
+				glog.Fatalf("Couldn't parse Nvidia GPUs: %v", err)
```

Comment:
Suggestion: "Couldn't parse -nvidia" to be more explicit that the problem is with the argument passed to `-nvidia`.

Reply:
Updated in the fixup https://github.com/livepeer/go-livepeer/pull/1840/commits/bbdb6499813206e5679181195a07701aa8673698